.travis.yml

```diff
@@ -4,24 +4,22 @@ language: python
 dist: bionic
 
 python:
-  - 3.7
+  - 3.6
```

Comment:
Can you explain the reason this was downgraded? Key dependencies not being compatible with Python 3.7, which has been out for over two years, is a significant worry given that Python 3.6 only has another year and a half to live.

Reply:
No idea, will check with 3.7

Reply:
Thanks!